backend/tests/test_services.py

```
@@ -1,201 +1,210 @@
-#     # Mock db session and Post class
-#     db = MagicMock()
+# Test for save_upload_file function
+def test_save_upload_file():
+    """Test the save_upload_file function"""
+    # Create mock file
+    mock_file = MagicMock()
+    upload_file = MagicMock(spec=UploadFile)
+    upload_file.file = mock_file
+    
+    # Path and filename
+    dest_dir = "/tmp/uploads"
```

Ruff (0.8.2)

Probable insecure usage of temporary file or directory: "/tmp/uploads"

(S108)